docs/api/index.md

```diff
@@ -46,7 +46,7 @@ app.listen(3000);
   control flows back "upstream".
 
   The following example responds with "Hello World", however first the request flows through
-  the `x-response-time` and `logging` middleware to mark when the request started, then continue
+  the `x-response-time` and `logging` middleware to mark when the request started, then continues
   to yield control through the response middleware. When a middleware invokes `next()`
```

Comment:
suggestion: This could be further simplified to "...mark when the request started, _then yields control_ through the response middleware."

Reply:
Even better, @8enSmith ! Thank you